### PR TITLE
Add some models

### PR DESCRIPTION
**Pull request by nmaac**
I add two models:
- FReLU
- WeightNet

### CODE REVIEW

File: models/megengine_vision_frelu.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+print(probs)
+```
+
+### 模型描述
+
+目前我们提供了部分在ImageNet上的预训练模型(见下表)，各个网络结构在ImageNet验证集上的表现如下：
+
+|        Model             | Activation | Top-1 err.|
+| :----------------------  | :--------: | :------:  |
+|    ResNet50              |  ReLU      | 24.0      |
```
Comment (FateScript) [inline models/megengine_vision_frelu.md:56]: why not provide top-5 error ?

File: models/megengine_vision_frelu.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+logits = model(processed_img)
+probs = F.softmax(logits)
+print(probs)
+```
+
+### 模型描述
+
+目前我们提供了部分在ImageNet上的预训练模型(见下表)，各个网络结构在ImageNet验证集上的表现如下：
+
+|        Model             | Activation | Top-1 err.|
```
Comment (FateScript) [inline models/megengine_vision_frelu.md:54]: @zhouyizhuang-megvii  should we use top-1 error or top-1 accuracy ?

**Review by FateScript — COMMENTED** [review]
PTAL @zhouyizhuang-megvii

File: models/megengine_vision_frelu.md (hunk 1 above)
Comment (nmaac) [reply models/megengine_vision_frelu.md:56]: Top-1 更能反应模型准确率，我们组的工作一直只比较 top-1，历史原因

File: models/megengine_vision_frelu.md (hunk 2 above)
Comment (zhouyizhuang-megvii) [reply models/megengine_vision_frelu.md:54]: top-1 error is okay

**Comment by zhouyizhuang-megvii** [comment]
> PTAL @zhouyizhuang-megvii

LGTM

File: models/megengine_vision_frelu.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+import cv2
+import numpy as np
+import megengine.data.transform as T
+import megengine.functional as F
+
+image = cv2.imread("cat.jpg").astype(np.float32)
+transform = T.Compose([
+    T.Resize(256),
+    T.CenterCrop(224),
+   # T.Normalize(mean=[103.530, 116.280, 123.675], std=[57.375, 57.120, 58.395]),  # BGR
```
Comment (xxr3376) [inline models/megengine_vision_frelu.md]: 既然无需 normlize，是不是这行直接删除比较好

File: models/megengine_vision_frelu.md (hunk 1 above)
Comment (xxr3376) [reply models/megengine_vision_frelu.md:56]: 提供 top-5 error 能更好的反应模型的性能，即便效果不佳也应该提供

File: models/megengine_vision_frelu.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
```
Comment (xxr3376) [inline models/megengine_vision_frelu.md]: 既然无需 Normlize，那么请不要提交没用的注释代码，请删除

File: models/megengine_vision_weightnet.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+import cv2
+import numpy as np
+import megengine.data.transform as T
+import megengine.functional as F
+
+image = cv2.imread("cat.jpg").astype(np.float32)
+transform = T.Compose([
+    T.Resize(256),
+    T.CenterCrop(224),
+   # T.Normalize(mean=[103.530, 116.280, 123.675], std=[57.375, 57.120, 58.395]),  # BGR
```
Comment (xxr3376) [inline models/megengine_vision_weightnet.md]: ditto

File: models/megengine_vision_weightnet.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
```
Comment (xxr3376) [inline models/megengine_vision_weightnet.md]: ditto

File: models/megengine_vision_frelu.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+    en_US: "Funnel Activation for Visual Recognition"
+    zh_CN: FReLU （ImageNet 预训练权重）
+author: MegEngine Team
+tags: [vision, classification]
+github-link: https://github.com/megvii-model/FunnelAct
+---
+
+```python
+import megengine.hub
+model = megengine.hub.load('megvii-model/funnelact', 'resnet50_frelu', pretrained=True)
```
Comment (xxr3376) [inline models/megengine_vision_frelu.md:14]: 如果还有 shufflenet_frelu 的话，最好加上第二行并处于注释状态

**Review by xxr3376 — CHANGES_REQUESTED** [review]

File: models/megengine_vision_frelu.md (hunk 1 above)
Comment (FateScript) [reply models/megengine_vision_frelu.md:56]: > 提供 top-5 error 能更好的反应模型的性能，即便效果不佳也应该提供

我猜是top-5的数值对model组意义不大？

Comment (xxr3376) [reply models/megengine_vision_frelu.md:56]: I don't know…但是我认为应该给用户更多的信息量才比较好…

Comment (nmaac) [reply models/megengine_vision_frelu.md:56]: Top-1 更能反应模型准确率，我们组的工作一直只比较 top-1
